easybuild/easyconfigs/b/BLAST+/BLAST+-2.6.0-foss-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,57 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>, Kenneth Hoste (UGent)
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'BLAST+'
+version = '2.6.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://blast.ncbi.nlm.nih.gov/'
+description = """Basic Local Alignment Search Tool, or BLAST, is an algorithm
+ for comparing primary biological sequence information, such as the amino-acid
+ sequences of different proteins or the nucleotides of DNA sequences."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'cstd': 'c++14'}
+
+sources = ['ncbi-blast-%(version)s+-src.tar.gz']
+source_urls = ['http://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/%(version)s/']
+
+patches = [
+    'BLAST+-2.2.30_ictce-fixes.patch',
+    'BLAST+-%(version)s_fix-make-install.patch',
+]
+
+checksums = ['0510e1d607d0fb4389eca50d434d5a0be787423b6850b3a4f315abc2ef19c996']
```

Comment:
@hajgato Please use `--include-checksums` to inject checksums for both `sources` & `patches`...

Reply:
@boegel Done!